### ISSUE TITLE
[BUG] how to fix worlds crash

### ISSUE THREAD

**Issue by Bexpg** [GrowtopiaNoobs/GrowtopiaServer#1262]
No description. am i 17 years old c# c++ programmer but i cant found where i doing wrong how to  fix worlds crash

**Comment by noobsteak** [comment]
Maybe u put too much mempcy in world generate?

**Comment by Bexpg** [comment]
no i dont

**Comment by Bexpg** [comment]
i think people using to gtbot to crash but gtbot crashing what

**Comment by Bexpg** [comment]
if no one gonna helping i gonna quit fff

**Comment by Bexpg** [comment]
I can't resist the 12 year old kids getting my server off

**Comment by playingoDEERUX** [comment]
yeah just quit thats what the whole world wants and its not gtbot.

**Comment by playingoDEERUX** [comment]
just quit ur pasta

**Comment by Bexpg** [comment]
ok am i quting dont dm me on discord for back or ignore

**Comment by Bexpg** [comment]
@playingoDEERUX  stfu no one asked i am asked questions how to fix server crash and you speaking about other shit

**Comment by playingoDEERUX** [comment]
before anyone will provide you a „fix“
you must change your behavior first and THEN you must at least learn to provide more information than just „how to fix server crash“

**Comment by Bexpg** [comment]
@playingoDEERUX  I want you to know that I do not respect anyone, I do not beg for any code, yes I ask questions, thats not mean i beg for code , I do not care about anyone i do not care f*cking privateserver i am doing everything for fun, I do not care about what others feel about me

**Comment by playingoDEERUX** [comment]
you may not care, thats fine its nothing deep like „feelings“ lmao, but you should still respect people cuz thats how stuff work in this world and if your parents just didnt explain u this then idk what to say other than just that its difficult to succeed without doing that first.

**Comment by Bexpg** [comment]
I'm just asking if they don't want to help , I don't care, I don't know how to fix the error, it doesn't mean I'm begging

**Comment by Bexpg** [comment]
do not forget that  "respect is given to deserving"

**Comment by playingoDEERUX** [comment]
yeah u didnt deserve the respect imo and you might not be begging directly in terms of „plz give code plz plz... and spamming etc.) but in general it is just stupid what you did because:
- Posting stuff that does not fit into the GrowtopiaServer repository
- Asking for fixes without giving any further information (no one will know how to help you if they dont know what exactly the problem is, and at least finding out where the problem is located at should be your part as you are the user reporting something.
- Generally being in my opinion dumb because you say stuff that makes no sense, or is just super weird and doesnt fit to some „17 year old“ like u.

**Comment by Bexpg** [comment]
"i dont care what other peoples saying"

**Comment by Bexpg** [comment]
and i cant give more info because i dont know to

**Comment by Bexpg** [comment]
i think people crashing worlds but not sure

**Comment by playingoDEERUX** [comment]
You can definitely get more info, by debugging or just using log files, if you know what the issue is then just apply a fix, and for that you need at least basic knowledge and theres always a reason for your program to crash, there is unexpected behavior or else which your program just doesnt like so it crashes. Either one of the public codes you used or because security checks fail or they are non existent or else.

**Comment by Bexpg** [comment]
How to debugging and also i using logs i cant found anything in logs file

**Comment by Bexpg** [comment]
i added captcha and its no pointed for me someone still can crash

**Comment by arshizx** [comment]
> No description. am i 17 years old c# c++ programmer but i cant found where i doing wrong How to fix server crash or worlds

c# c++ programmer must know how to fix it. if you can't, you are not a programmer

**Comment by cmdocmd** [comment]
@Bexpg you dont beg for codes ? nice joke

**Comment by JadlionHD** [comment]
> > No description. am i 17 years old c# c++ programmer but i cant found where i doing wrong How to fix server crash or worlds
> 
> c# c++ programmer must know how to fix it. if you can't, you are not a programmer

that's true, most of programmer using their logic to solve their problem. If they still dont know how to fix it, maybe they will asked to an forum or github repo that related with that issue.
